examples/guess-the-sketch/matchmaker/director_k8s.yaml

```diff
@@ -33,11 +33,18 @@ spec:
         - name: guess-the-sketch-director
           image: guess-the-sketch-director
           imagePullPolicy: Always
+          env:
+          - name: OM_CORE_ADDRESS
+            value: "https://om-core-976869741551.us-central1.run.app"
```

Comment:
This looks like a hard coded address. I assume it will not work outside your project? Can we dynamically pass it in?

Reply:
I can't think of an easy way to find this URL since the the user has to deploy OM2 on Cloud Run themselves(unless there is?) 

As a remedy, I've changed the url to be `YOUR_OM_CORE_URL` and modified the README.md of the Guess-the-Sketch demo to include the step where they should replace this value to be the url of the OM2 deployed on Cloud Run.